modules/HapticGlove_module/src/LinearRegression.cpp

```diff
@@ -0,0 +1,61 @@
+#include <LinearRegression.hpp>
+
+using namespace HapticGlove;
+
+LinearRegression::LinearRegression()
+{
+}
+
+LinearRegression::~LinearRegression()
+{
+}
```

Comment:
In general, if you need to specify a custom destructor, you would need to specify also the custom copy/move constructor and copy/move assignment. This is the so called [``rul of five``](https://en.cppreference.com/w/cpp/language/rule_of_three). In general, if you can avoid specifying a custom destructor, it is better.

Sometimes the destructor is needed when using this object inside a smart pointer. I am not sure if this is the case.

Reply:
Thanks for the suggestion. I set all the destructors to `default`.